### PR TITLE
Nette Frameworks in it's config file which generates DI Container is …

### PR DESCRIPTION
**Pull request by mkoula**
Nette Frameworks (nette.org) in it's config file which generates DI Container from (neon file - similar to yaml) is using annotations which are missing here. I hope it would not be an issue to have them here.

### CODE REVIEW

File: lib/Thumbor/Url/BuilderFactory.php [hunk 1]
```diff
@@ -24,17 +24,31 @@ class BuilderFactory
     private $server;
     private $secret;
 
+    /**
+     * @param $server
+     * @param null $secret
```
Comment (arthens) [inline lib/Thumbor/Url/BuilderFactory.php]: shouldn't this be `string|null`?

File: lib/Thumbor/Url/BuilderFactory.php [hunk 2]
```diff
@@ -24,17 +24,31 @@ class BuilderFactory
...
+     * @return BuilderFactory
+     */
     public static function construct($server, $secret=null)
     {
         return new self($server, $secret);
     }
 
+    /**
+     * BuilderFactory constructor.
```
Comment (navitronic) [inline lib/Thumbor/Url/BuilderFactory.php]: This kind of comment is probably unnecessary

**Comment by mkoula** [comment]
I cleaned the annotations according to the comments

**Comment by arthens** [comment]
Looks good, thanks for the PR

**Pull request merged**